CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ Note: We're not following semantic versioning yet, we are going to talk about th
 
 Breaking changes:
```

Comment:
ideally the change doesn't include empty placeholders. we'd  stop at `## 0.0.27-alpha (Breaking release)`

Reply:
the rest seems to be as expected

Reply:
I figured putting in placeholders might be helpful for the next contributors but actually it makes the file harder to read so I've removed those. Thanks @igloosi.